### PR TITLE
Cleanup script for jaeger removal

### PR DESCRIPTION
**Pull request by hisarbalik**
**Description**

Changes proposed in this pull request:

- Shell script to cleanup remaining resources after Jaeger tracing removal

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/16584

### CODE REVIEW

**Review by NHingerl — COMMENTED** [review]
as a TW, I cannot judge whether this script does what it should. However, I can approve if you need to merge.

**Review by NHingerl — APPROVED** [review]

File: docs/assets/2.13.-2.14-cleanup-jaeger.sh
Comment (a-thaler) [inline docs/assets/2.13.-2.14-cleanup-jaeger.sh:4]: Why no review happened here from a developer?
The resource is not placed in the kyma-system namespace and the command will fail, see https://github.com/kyma-project/kyma/blob/2.13.0/resources/tracing/templates/kyma-additions/istio-telemetry.yaml#L6

Why cluster scoped resources are using a namespace in he script?
Also the migration guide related to the script is missing, see for example https://kyma-project.io/docs/kyma/2.12/migration-guide-2.11-2.12

Comment (NHingerl) [reply docs/assets/2.13.-2.14-cleanup-jaeger.sh:4]: I, too, was wondering why the only mandatory review was by TW. 
Is the code owners assignment wrong?

**Pull request merged**
